src/style.css

```diff
@@ -1,3 +1,81 @@
+html{
+  font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  height: 60px;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: auto 0;
+}
+
+.navbar {
+  display: flex;
+  max-height: 60px;
+  align-items: center;
+}
+
+.list{
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  list-style: none;
+  min-height: 60px;
+  align-items: center;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: black;
+  line-height: 60px;
+  display: grid;
+}
+
+.nav-margin {
+  margin: 0 10px;
+}
+
+.nav-link-1 {
+  margin: 0 10px 0 0;
+}
+
+.nav-link-8 {
+  margin: 0 0 0 10px;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
The color `#00acdc` is used directly here. Consider defining it as a CSS variable for consistency and easier maintenance. For example: `--primary-blue: #00acdc;` and then use `color: var(--primary-blue);`.